### DISCUSSION TITLE
Monorepo doesn't work with custom port.

### DISCUSSION THREAD

**Discussion by cosbgn** [vercel/vercel#4130] [General]
I have a `nuxt` app and an `/api` backend. This works ok when I run `now dev` it starts on port `3000` and proxies api calls to the functions. However recently my nuxt app has become really slow, the only solution to this is to use a different port at each time. So on my nuxt app I have this:

```
const randomPort = Math.floor(Math.random() * 8000) + 1025  
export default {
	server: {
		port: randomPort, // default: 3000
		host: 'localhost' // default: localhost
	}
      // .... everything else
}
```
This finally works, my nuxt app is again fast, and I can develop without issues, however now.sh doesn't proxies anymore my functions, If I go on `localhost:3453` (or whatever the new random port is) I don't get my api functions, but only nuxt 404 page.

Why `now dev` doesn't pickup my random port? Any suggestions?

**Comment by TooTallNate** [comment]
Hey there @Cosbgn, thanks for the question. If you could share an example repository that demonstrates your issue, then I'll take a closer look and try to understand what's happening.

**Reply by cosbgn** [reply]
Hey @TooTallNate I've created a minimal repo: https://gitlab.com/cosbgn/proxy-vercel-port-error

Simply `npm i` and then if you change the first line (`useRandomPort`) on `nuxt.config.js` from `false` to `true` the backend proxy won't work anymore.

Let me know if something is not clear. 
Costantin

**Reply by cosbgn** [reply]
Hey @TooTallNate any update on this?

**Reply by TooTallNate** [reply]
Hey @Cosbgn, thanks for the repo! I'll be taking a look at it this week.

**Reply by amine-abdelli** [reply]
> Hey @Cosbgn, thanks for the repo! I'll be taking a look at it this week.

Sweet, any news about this ?
